api.js

```diff
@@ -0,0 +1,106 @@
+// use of API level -350
+
+const header = document.getElementById("header");
+const searchBox = document.querySelector("#search-shows");
+const paragraphforSearch = document.querySelector("#paragraphforSearch");
+const content = document.querySelector(".content");
+const selectShow = document.querySelector("#select-show");
+const selectEpisode = document.querySelector("#select-episode");
+let allEpisodes;
+const allShows = getAllShows();
+// console.log(title)
+
+loadShowOnPage(allShows);
+// popping up the episode in the window while it loads
+function loadShowOnPage(ShowList) {
+  header.innerHTML = "";
+  ShowList.sort((a, b) => a.name.localeCompare(b.name));
+  loadShow(ShowList);
+}
+
+// creating function for shows
+function loadShow(shows) {
+  for (let show of shows) {
+    const title = document.createElement("h2");
+    const image = document.createElement("img");
+    const paragraph = document.createElement("p");
+    const titleDiv = document.createElement("div");
+    const newbody = document.createElement("div");
+    //  adding episod name
+    title.innerText = `${show.name}`;
+    // title.appendChild(span)

```

Comment:
remember to remove bits of code you are no longer using, it just makes your code easier for other to read :)